### ISSUE TITLE
“(data, label) in enumerate(train_datait)” error! 

### ISSUE THREAD

**Issue by xiaowenhe** [linmx0130/ya_mxdet#6]
Thanks for your work! 
When I follow your work, I get a error. 
In train_faster_rcnn.py:
line 23:  train_datait = mx.gluon.data.DataLoader(train_dataset, batch_size=1, shuffle=True). 
 Here batch_size = 1, no error. 
But change batch_size to other greater than 1,   
line 37:  (data, label) in enumerate(train_datait)  give a error!
How to solve it ? Thank you very much.

**Comment by linmx0130** [comment]
I am sorry that until now ya_mxdet does not support batch_size > 1. For detection, training with batch_size > 1 is much complex, since the program should handle different size of pictures.

**Comment by xiaowenhe** [comment]
OK, thank you very much! In mxnet API, A MultiProposal function is supply for getting proposals. But in your code,you get it by yourself code? Will you update your code using mxnet API,such as MultiProposal,MultiBoxTarget and so on?  Thank you !

**Comment by linmx0130** [comment]
I will try it in late Febuary. Thank you for informing me!
